src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -253,6 +253,17 @@ def is_port_internal(port_name, namespace=None):
     return False
 
 
+def get_external_ports(port_names, namespace=None):
+    external_ports = set()
+    ports_config = get_port_table(namespace)
+    for port in port_names:
+        if port in ports_config:
+            if (PORT_ROLE not in ports_config[port] or
+                ports_config[port][PORT_ROLE] == EXTERNAL_PORT):
+                external_ports.add(port)
```

Comment:
This is unfortunate alignment and violates PEP8 [E129](https://www.flake8rules.com/rules/E129.html).

A tool like autopep8 will indent the line 4 more spaces to fix. E.g.,

```
            if (PORT_ROLE not in ports_config[port] or
                    ports_config[port][PORT_ROLE] == EXTERNAL_PORT):
                external_ports.add(port)
```

Reply:
that's right, I should install pep8 package to my editor.